### PR TITLE
Setup basic Node project with Express and TypeScript

### LINKED ISSUE #1: Move code here
Move setup from https://github.com/vega/editor/tree/server

### PR DESCRIPTION
**Pull request by algomaster99**
I have set up the project with basic configurations. All it has now are packages installed and a simple command to start the server. I have updated `README.md` as well. 

Fixes #1

### CODE REVIEW

**Comment by siddhant1** [comment]
Are we sure about the selection of typescript here?

**Comment by siddhant1** [comment]
Also , provide a dummy port if the process.env.PORT does not have any value

**Comment by algomaster99** [comment]
@siddhant1 Is there some problem in using TypeScript? I used it for two reasons.
-  I referred to [this link](https://www.geeksforgeeks.org/difference-between-typescript-and-javascript/) and found that there are a few advantages of using it.
-  Most projects in Vega are based on TypeScript so it maintains uniformity.

**Comment by domoritz** [comment]
Using typescript is great as it helps eliminate common bugs and enables better autocomplete.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "vega-editor-backend",
+  "version": "1.0.0",
```
Comment (domoritz) [inline package.json]: Let's start with pre 1.0 for now.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  },
+  "author": "",
+  "bugs": {
+    "url": "https://github.com/vega/editor-backend/issues"
+  },
+  "homepage": "https://github.com/vega/editor-backend#readme",
+  "devDependencies": {
+    "@types/express": "^4.16.1",
+    "tslint": "^5.16.0",
+    "typescript": "^3.4.5"
```
Comment (domoritz) [inline package.json]: Please use eslint as tslint is deprecated. See Vega-Lite and Vega Embed.

File: package.json (hunk 1 above)
Comment (algomaster99) [reply package.json]: @domoritz I think we should follow the [semantic versioning guidelines](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase). Shall I rewrite it to 0.1.0?

Comment (domoritz) [reply package.json]: Yes, we do follow it everywhere. 0.1 sounds good.

**Comment by algomaster99** [comment]
@domoritz I have made some changes. Please review it again.

**Pull request merged**